tests/baselines/reference/propertyAccess3.errors.txt

```diff
@@ -1,8 +1,10 @@
 tests/cases/compiler/propertyAccess3.ts(2,5): error TS2339: Property 'toBAZ' does not exist on type 'boolean'.
+  Property 'toBAZ' does not exist on type 'true'.
 
 
 ==== tests/cases/compiler/propertyAccess3.ts (1 errors) ====
     var foo: boolean;
     foo.toBAZ();
         ~~~~~
-!!! error TS2339: Property 'toBAZ' does not exist on type 'boolean'.
\ No newline at end of file
+!!! error TS2339: Property 'toBAZ' does not exist on type 'boolean'.
+!!! error TS2339:   Property 'toBAZ' does not exist on type 'true'.
```

Comment:
This looks weird - let's exclude `TypeFlags.Boolean` along with `TypeFlags.Enum`